unitTests/ui/MLFacet/MLFacetValues/MLFacetValueTest.ts

```diff
@@ -60,9 +66,37 @@ export function MLFacetValueTest() {
       expect(mLFacetValue.equals(value)).toBe(true);
     });
 
-    // TODO: tests
-    // formattedCount
-    // valueCaption
-    // render (only check that is doesn't throw)
+    it(`when getting formattedCount with a non zero value
+      it should return a string in the Globalize format`, () => {
+      expect(mLFacetValue.formattedCount).toBe(Globalize.format(mLFacetValue.numberOfResults, 'n0'));
+    });
+
+    it(`when getting formattedCount with a zero value
+      it should return an empty string`, () => {
+      mLFacetValue.numberOfResults = 0;
+      expect(mLFacetValue.formattedCount).toBe('');
+    });
+
+    it(`when using the valueCaption option with a function
+      should bypass it and return the original value`, () => {
+      initializeComponent({ valueCaption: () => 'allo' });
```

Comment:
A better pattern would be to write your options to a variable on ~line 11, and then have `initializeComponent` read that variable. This allows you to sequentially build your options in nested `describe`/`it` blocks and keeps the `initalizeComponent()` line short. e.g.

```
let options: IMLFacetOptions;

it(`uses the valueCaption`, () => {
    option.valueCaption = { key: 'value' };
    initComponent();

    expect(mlFacetValue.valueCaption).toBe(options.valueCaption.key)
})
```